tcms/testruns/forms.py

```diff
@@ -339,7 +339,7 @@ def clean_case_text_version(self):
     def clean_case_run_status(self):
         data = self.cleaned_data.get('case_run_status')
         if not data:
-            data = TestCaseRunStatus.get_IDLE()
+            data = TestCaseRunStatus.get_idle()
```

Comment:
This method is used only here, also looking at the method it does some weird caching that we don't really need. 

Drop this method entirely and just use the query here.